rules_jvm_export/jvm_export/support/deploy.py

```diff
@@ -0,0 +1,264 @@
+#!$PYTHON_PATH
+
+# Originally from https://github.com/vaticle/bazel-distribution/blob/master/maven/templates/deploy.py
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import print_function
+from xml.etree import ElementTree
+
+import argparse
+import hashlib
+import os
+import re
+import shutil
+import subprocess as sp
+import sys
+import tempfile
+from pathlib import Path
+from posixpath import join as urljoin
+from urllib.parse import urlparse
+
+USAGE_STR = """
+bazel run <target>.publish -- release
+"""
+
+MAVEN_REPOS = {
+    "snapshot": "{snapshot}",
+    "release": "{release}",
+    "local": "file:~/.m2/repository/",
+}
+
+
+def main():
+    args = _parse_args()
+    repo_type = "local" if args.local else args.command
+    maven_url = MAVEN_REPOS[repo_type]
+    jar_path = "$JAR_PATH"
+    pom_file_path = "$POM_PATH"
+    srcjar_path = "$SRCJAR_PATH"
+
+    namespaces = {"ns0": "http://maven.apache.org/POM/4.0.0"}
```

Comment:
I think this is unused

Reply:
Yea. I forgot to remove that after refactoring to `_parse_pom()`